src/OrchardCore.Modules/OrchardCore.Users/Views/UserMenu.cshtml

```diff
@@ -1,15 +1,17 @@
 @inject IAuthorizationService AuthorizationService
-<form asp-route-area="OrchardCore.Users" asp-controller="Account" asp-action="LogOff" method="post" class="form-inline">
-    <div class="dropdown">
-        <button class="btn btn-sm btn-light dropdown-toggle" type="button" id="dropdownMenu2" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-            <span><i class="fa fa-user" aria-hidden="true"></i></span>&nbsp;<span>@User.Identity.Name</span>
-        </button>      
-        <div class="dropdown-menu dropdown-menu-right w-25">
+<div class="dropdown">
+    <form asp-route-area="OrchardCore.Users" asp-controller="Account" asp-action="LogOff" method="post" class="form-inline">
+        <a class="nav-link dropdown-toggle" id="navbarDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            <span><i class="fa fa-user" aria-hidden="true"></i>
+        </a>
+        <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
+            <h6 class="dropdown-header">@User.Identity.Name</h6>
```

Comment:
```suggestion
            <h6 class="dropdown-header">@T["Signed in as"]<span class="ml-1 font-weight-bold">@User.Identity.Name</span></h6>
```

Reply:
Not sure for this one.

Reply:
Not necessary but it is consistent with what Github does here :  

![image](https://user-images.githubusercontent.com/3228637/102024954-e6035700-3d62-11eb-81a2-069bc9e6a145.png)